src/ajax/map_variants.php

```diff
@@ -309,9 +309,38 @@ function lovd_mapVariantToTranscripts (&$aVariant, $aTranscripts)
                         continue;
                     }
 
-                    // Great! All checks have passed and the new description
-                    //  can be sent to the database.
-                    $aNewVariant = lovd_getVariantInfo($aVVResponse['data'][$sBuild]);
+                    // Great! All checks have passed. Now all that is left to do is to
+                    //  ensure the variants are correctly concatenated in case multiple
+                    //  descriptions were found on the build in question.
+                    if (is_string($aVVResponse['data'][$sBuild])) {
```

Comment:
This whole thing (line 315-338) is also performed in `ajax/check_hgvs_dialogue.php` in the branch `improve/variantCreationForm`. Perhaps we could unify the code by building a function for both cases?

Reply:
In that case, that should then be in the VV library. I'm not really sure yet if maybe this entire problem should be solved inside of the VV library instead of outside of it.

Reply:
I think it would be nice to do so inside the library!